net.veloren.airshipper.yaml

```
@@ -14,29 +11,64 @@ finish-args:
   - --share=ipc
   - --device=dri
 build-options : {
-  append-path : "/usr/lib/sdk/rust-nightly/bin"
+  append-path : "/app/sdk/rust-nightly/bin"
 }
+cleanup:
+  - /sdk
 
 modules:
+  - name: rust
+    buildsystem: simple
+    sources:
+      - type: archive
+        only-arches:
+          - aarch64
+        dest: rust
+        url: https://static.rust-lang.org/dist/2024-01-15/rust-nightly-aarch64-unknown-linux-gnu.tar.xz
+        sha256: 946ca47cb7797a94abf4b83b3410d62f17e47dca2bf1841b75ce36bcde72bb10
+        x-checker-data:
+          type: rust
+          package: rust
+          channel: nightly
+          target: aarch64-unknown-linux-gnu
+      - type: archive
+        only-arches:
+          - x86_64
+        dest: rust
+        url: https://static.rust-lang.org/dist/2024-01-15/rust-nightly-x86_64-unknown-linux-gnu.tar.xz
+        sha256: 6ae03811db0b812220fc5abec286fa41a51f946163f38e804820717b566f8e22
+        x-checker-data:
+          type: rust
+          package: rust
+          channel: nightly
+          target: x86_64-unknown-linux-gnu
+    build-commands:
+      - cd rust && ./install.sh --prefix=/usr/lib/sdk/rust-nightly --without=rust-docs
+        --without=rust-docs-json-preview --disable-ldconfig --verbose
+      - cd rust-src && ./install.sh --prefix=/usr/lib/sdk/rust-nightly --disable-ldconfig
+        --verbose
```

```suggestion
      - cd rust && ./install.sh --prefix=/app/sdk/rust-nightly --without=rust-docs
        --without=rust-docs-json-preview --disable-ldconfig --verbose
```

Command is same for both, `/usr` prefix is from runtime it is read only.